railties/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Add `rails db:system` command to display an app's DBMS for a given environment.
+
+    ```
+    bin/rails db:system -e production
+    postgresql
```

Comment:
@dhh how badly do we actually need this `db:system` inquiry command? The output doesn't seem that useful, especially when split by environment. And if not split by environment then wouldn't people be just as well off reading `config/database.yml`?

Seems like `db:system:change` (or whatever we're calling it, since I don't remember adding such deep nesting to the commands) is the real meat.